src/ResourceManager/Profile/Commands.Profile/AzureRmAlias/DisableAzureRmAlias.cs

```diff
@@ -0,0 +1,188 @@
+                            SessionState.PSVariable.Remove("Alias:" + modulemapping[name]);
+                            if (PassThru)
+                            {
+                                WriteObject("Removed: " + modulemapping[name] + " : " + name);
+                            }
+                        }
+                    }
+                }
+            }
+            else
+            {
+                foreach (var module in Module)
+                {
+                    if (mapping.ContainsKey(module))
+                    {
+                        Dictionary<string, string> modulemapping =
+                            (Dictionary<string, string>)JsonConvert.DeserializeObject(mapping[module].ToString(), typeof(Dictionary<string, string>));
+                        foreach (var name in modulemapping.Keys)
+                        {
+                            // For every alias, remove pairing in the Alias provider if it exists
+                            if (SessionState.PSVariable.GetValue("Alias:" + modulemapping[name]) != null)
+                            {
+                                SessionState.PSVariable.Remove("Alias:" + modulemapping[name]);
+                                if (PassThru)
+                                {
+                                    WriteObject("Removed: " + modulemapping[name] + " : " + name);
+                                }
+                            }
+                        }
+                    }
+                    else
+                    {
+                        WriteWarning("Module '" + module + "' is not a valid Az module.");
+                    }
+                }
+            }
+
+            // Set path to user profile based on the Scope
+            object userprofile = "";
+            if (Scope != null && Scope.Equals("CurrentUser"))
+            {
+                userprofile = SessionState.PSVariable.GetValue("env:USERPROFILE") + "\\Documents\\PowerShell\\profile.ps1";
+            }
+
+            else if (Scope != null && Scope.Equals("LocalMachine"))
+            {
+                userprofile = SessionState.PSVariable.GetValue("PSHOME") + "\\profile.ps1";
+            }
+
+            if (Scope != null && (Scope.Equals("CurrentUser") || Scope.Equals("LocalMachine")))
+            {
+                if (File.Exists(userprofile.ToString()))
+                {
+                    List<string> modulesToKeep = new List<String>();
+                    string filecontent = "";
+                    string originalText = File.ReadAllText(userprofile.ToString());
+                    if (originalText.Contains("#Begin Azure PowerShell alias import"))
+                    {
+                        // Add back profile code unrelated to Azure PowerShell aliases
+                        var splitOriginalText = originalText.Split(new string[] { "#Begin Azure PowerShell alias import", "#End Azure PowerShell alias import" }, StringSplitOptions.None);
```

Comment:
let's make these marker comments as constants, so we can reuse them, and would only need to change then in one place